sdk/core/src/services/sboms/service.rs

```diff
@@ -113,13 +114,16 @@ impl SbomService {
             }
         };
 
+        // Ensure the display timestamp is set if successful.
+        sbom.created = platform::time::iso8601_timestamp()?;
```

Comment:
Thanks for adding this!  Same comment though about it being an option or else needing a migration.

Reply:
It is an option

Reply:
Already that is, I changed the signature of the function